src/Login/Login.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import { useRecoilState, useRecoilValue } from 'recoil';
+import { userAtom } from './atoms';
+import './Login.css'
+const LoginForm = () => {
+    const {
+        register,
+        handleSubmit,
+        formState: { errors },
+    } = useForm({
+        mode: 'onChange',
```

Comment:
`Low priority`

come with a performance compromsise 

so add only if necessary

onChange | string | Validation is triggered on the changeevent for each input, leading to multiple re-renders. Warning: this often comes with a significant impact on performance.
-- | -- | --